dnf-behave-tests/dnf/offline.feature

```diff
@@ -257,3 +257,16 @@ Given I use repository "simple-base"
   And Transaction is following
       | Action        | Package                       |
       | install       | labirinto-0:1.0-1.fc29.x86_64 |
+
+
+# https://github.com/rpm-software-management/dnf5/issues/1851
+Scenario: Test offline install and offline status
```

Comment:
I think the scenario name is a left over from the copy.

Reply:
Shame on me! Fixed.